__tests__/WinnerPageTest.js

```diff
@@ -0,0 +1,10 @@
+import { WinnerScreen } from '../components/WinnerScreen'
+import React from 'react'
+import renderer from 'react-test-renderer'
+
+describe("WinnerScreen component", () => {
```

Comment:
inconsistent quotation marks; you use single quotes everywhere else

Reply:
Fixed it @squidapillar